tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -889,4 +902,89 @@ public function theFollowingUsersShouldBeListedInFollowingGroups(TableNode $tabl
 			);
 		}
 	}
+
+	/**
+	 * @param string $user
+	 *
+	 * @return ResponseInterface
+	 * @throws JsonException
+	 * @throws GuzzleException
+	 */
+	public function retrieveUserInformationUsingGraphApi(
+		string $user
+	):ResponseInterface {
+		$credentials = $this->getAdminOrUserCredentials($user);
+		return GraphHelper::getOwnInformationWithGroups(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			$credentials["username"],
+			$credentials["password"],
+		);
+	}
+
+	/**
+	 * @When /^the user "([^"]*)" retrieves (:?her|his) information using the Graph API$/
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 * @throws JsonException
+	 */
+	public function userRetrievesHisorHerInformationOfUserUsingGraphApi(
+		string $user
+	):void {
+		$response = $this->retrieveUserInformationUsingGraphApi($user);
+		$this->featureContext->setResponse($response);
+	}
+
+	/**
+	 * @Then /^the api response should contain the following information:$/
+	 *
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 */
+	public function theApiResponseForUserShouldContainTheFollowingInformation(TableNode $table): void {
+		$rows = $table->getRowsHash();
+		$apiResponse = $this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse());
+
+		foreach (array_keys($rows) as $keyName) {
+			if ($keyName === 'memberOf') {
+				$memberOfFromApiReponse = [];
+				$memberOf = preg_split('/\s*,\s*/', trim($rows['memberOf']));
+				foreach ($apiResponse['memberOf'] as $member) {
+					$memberOfFromApiReponse[] = $member['displayName'];
+				}
+				Assert::assertEqualsCanonicalizing($memberOf, $memberOfFromApiReponse);
+			} elseif ($keyName === 'id') {
+				// check if the user_if from response is in format UUIDv4
```

Comment:
could be simply this
```suggestion
				// check user id
```